source/api/injector.md

```diff
@@ -115,6 +114,7 @@ hexo.extend.injector.register('head_end', require('./lib/inject')(hexo))
 
 ``` js lib/inject.js
 module.exports = (hexo) => () => {
+  const css = hexo.extend.helper.get('css');
```

Comment:
just tested this line and it doesn't work due to `this.url_for()`.

https://github.com/hexojs/hexo/blob/14890743dd00e1a1e7c4bd9c7ef632f609d35a4c/lib/plugins/helper/css.js#L27

Reply:
const css = hexo.extend.helper.get('css').bind(hexo) doesn't work for me either, tried v4.0.0, v4.1.1, v4.2.1, v5.

Reply:
https://github.com/hexojs/hexo/pull/4447

Reply:
Fix released in [5.0.1](https://github.com/hexojs/hexo/releases/tag/5.0.1)